src/main/java/com/endava/Ticket_Management_System/model/Orders.java

```diff
@@ -0,0 +1,93 @@
+package com.endava.Ticket_Management_System.model;
+
+import jakarta.persistence.*;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+@Entity
+@Table(name = "Orders")
+public class Orders implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int orderID;
+
+    @ManyToOne
+    @JoinColumn(name = "customerID")
+    private Customer customerID;
+
+    @ManyToOne
+    @JoinColumn(name = "ticketCategoryID")
+    private TicketCategory ticketCategoryID;
+
+    @Column(name = "orderedAt")
+    private Date orderedAt;
+
+    @Column(name = "numberOfTickets")
+    private int numberOfTickets;
+
+    @Column(name = "totalPrice")
+    private BigDecimal totalPrice;
+
+    public Orders() {
```

Comment:
Why didn't you use lombok for constructors, getter and setters? I see you have it as a dependency